src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -773,10 +773,17 @@ private bool CheckEventValueKind(BoundEventAccess boundEvent, BindValueKind valu
 
                 if (receiver?.Kind == BoundKind.BaseReference && eventSymbol.IsAbstract)
                 {
-                    Error(diagnostics, ErrorCode.ERR_AbstractBaseCall, boundEvent.Syntax, eventSymbol);
-                    return false;
+                    var baseReference = (BoundBaseReference)receiver;
+
+                    if (!eventSymbol.IsImplementableInterfaceMember() ||
```

Comment:
>IsImplementableInterfaceMember [](start = 37, length = 30)

Just as you can use a base invocation to a non-virtual member in a class, base invocations to an interface do not require that the symbol found is virtual or abstract.

Reply:
> Just as you can use a base invocation to a non-virtual member in a class, base invocations to an interface do not require that the symbol found is virtual or abstract.

Completely agree and tests confirm that.

---
In reply to: [255762096](https://github.com/dotnet/roslyn/pull/33281#discussion_r255762096) [](ancestors = 255762096)